README.md

```diff
@@ -831,6 +831,7 @@ Instructs the module to ignore a specified rule ID. Note that ignoring a rule in
 	location / {
 		access_by_lua '
 			waf:set_option("ignore_rule", 40294)
+			waf:set_option("ignore_rule", {40002, 41036})
```

Comment:
This change seems unnecessary? We have many options that can accept a table as the second arg, and don't provide examples for both usages. However, I won't block on this.